src/Bridges/ApplicationLatte/TemplateFactory.php

```diff
@@ -94,6 +94,9 @@ public function createTemplate(UI\Control $control = NULL)
 		$template->flashes = [];
 		$latte->addProvider('uiControl', $control);
 		$latte->addProvider('uiPresenter', $presenter);
+		if ($control) {
+			$latte->addProvider('snippetBridge', new Nette\Bridges\ApplicationLatte\SnippetBridge($control));
```

Comment:
IMHO `$latte->addProvider('snippetDriver', new Latte\SnippetDriver(Nette\Bridges\ApplicationLatte\SnippetBridge($control)));` is fine.